### DISCUSSION TITLE
Insecurly

### DISCUSSION THREAD

**Discussion by zek-c** [zek-c/Securly-Kill-V111#80] [General]
Only things about insecurly exploit

**Comment by Gigibamboo** [comment]
It doesn't work.

**Reply by zek-c** [reply]
yep its been patched ok

**Comment by Bruhsbegood128943** [comment]
It wont work on my Chromebook for some reason. Is that because my Chromebook is version 121 or no?

**Reply by zek-c** [reply]
nah it just don't work anymore

**Reply by OrangeThunderFire** [reply]
It works for me, but not my friend. Can you make a new one

**Reply by zek-c** [reply]
> It works for me, but not my friend. Can you make a new one

no? i will not make another... not that easy, it's simply a version diff between u and ur friend

**Comment by OrangeThunderFire** [comment]
bruh
